cs251tk/student/markdownify/process_file.py

```diff
@@ -43,6 +52,12 @@ def compile_file(filename, steps, results, supporting_dir):
             'status': status,
         })
 
+        if web:
+            if status == 'success':
+                input("{} - {}".format(student, filename))
```

Comment:
Why is this collecting input?

Reply:
It's waiting for the user to move on to the next file. It doesn't do anything differently if the user actually enters something.

Reply:
Ah, ok.